.github/workflows/asan.yml

```
@@ -0,0 +1,182 @@
+    - name: Run tests under ASan
+      working-directory: ${{github.workspace}}/build/OloEngine/tests
+      run: ctest -C Debug -V
+      env:
+        ASAN_OPTIONS: halt_on_error=1
+
+    - name: Upload test results
+      if: always()
+      uses: actions/upload-artifact@v7
+      with:
+        name: asan_test_results_windows
+        path: ${{runner.workspace}}/OloEngineBase/test_results/*.xml
```

actionlint (1.7.12)

[error] property "workspace" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)